pkgs/development/compilers/jetbrains-jdk/prebuild.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, openjdk11, fetchurl, jetbrains, stdenv }:
+
+openjdk11.overrideAttrs (oldAttrs: rec {
```

Comment:
Why do an override of the (source build?) openjdk expression? Just use `stdenvNoCC.mkDerivation`.